src/devices/Bmp280/Bmp280.cs

```diff
@@ -0,0 +1,592 @@
+            CalibrationData = new Bmp280CalibrationData();
+            protocol = CommunicationProtocol.I2c;
+        }
+
+        public Bmp280(I2cConnectionSettings settings)
+        {
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            {
+                _i2cDevice = new Windows10I2cDevice(settings);
+            }
+            else if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
+            {
+                _i2cDevice = new UnixI2cDevice(settings);
+            }
+            else
+            {
+                throw new NotImplementedException();
+            }
+            protocol = CommunicationProtocol.I2c;
+            CalibrationData = new Bmp280CalibrationData();
+        }
+
+        private void Begin()
+        {
+            byte[] readBuffer = new byte[] { 0xFF };
+
+            _i2cDevice.WriteByte((byte)Registers.REGISTER_CHIPID);
+            _i2cDevice.Read(readBuffer);
+
+            if (readBuffer[0] != Signature)
+            {
+                return;
+            }
+            initialised = true;
+
+            //Read the coefficients table
+            ReadCoefficients();
+        }
+
+        /// <summary>
+        ///  Reads the factory-set coefficients 
+        /// </summary>
+        private void ReadCoefficients()
+        {
+            // Read temperature calibration data
+            CalibrationData.DigT1 = Read16((byte)Registers.REGISTER_DIG_T1);
+            CalibrationData.DigT2 = (short)Read16((byte)Registers.REGISTER_DIG_T2);
+            CalibrationData.DigT3 = (short)Read16((byte)Registers.REGISTER_DIG_T3);
+
+            // Read presure calibration data
+            CalibrationData.DigP1 = Read16((byte)Registers.REGISTER_DIG_P1);
+            CalibrationData.DigP2 = (short)Read16((byte)Registers.REGISTER_DIG_P2);
+            CalibrationData.DigP3 = (short)Read16((byte)Registers.REGISTER_DIG_P3);
+            CalibrationData.DigP4 = (short)Read16((byte)Registers.REGISTER_DIG_P4);
+            CalibrationData.DigP5 = (short)Read16((byte)Registers.REGISTER_DIG_P5);
+            CalibrationData.DigP6 = (short)Read16((byte)Registers.REGISTER_DIG_P6);
+            CalibrationData.DigP7 = (short)Read16((byte)Registers.REGISTER_DIG_P7);
+            CalibrationData.DigP8 = (short)Read16((byte)Registers.REGISTER_DIG_P8);
+            CalibrationData.DigP9 = (short)Read16((byte)Registers.REGISTER_DIG_P9);
+
+        }
+
+        /// <summary>
+        /// Set mode sleep (no measurements)
+        /// </summary>
+        public void SetModeSleep()
+        {
+            byte status = Read8((byte)Registers.REGISTER_CONTROL);
+            //clear last two bits
+            status = (byte)(status & 0b11111100);
+            _i2cDevice.Write(new byte[] { (byte)Registers.REGISTER_CONTROL, status });
+        }
+
+        /// <summary>
+        /// Sets mode to forced (device goes to sleep after one reading)
+        /// </summary>
+        public void SetModeForced()
```

Comment:
Is this how the mode is called in the documentation? I just ask because it is not super clear to me what forced means until I read your comment.

Reply:
Thanks for all feedback!

It is called forced mode everywhere in the datasheet:
> BMP280 can be operated in three power modes (see chapter 3.6):
> sleep mode
> normal mode
> forced mode

Reply:
I see, in that case this is fine. Having the summary comments helps a bunch, since consumers that don't know what it means (like me :smile:) will be able to know what is the use for it.